dcicutils/secrets_utils.py

```diff
@@ -20,14 +21,18 @@
 GLOBAL_APPLICATION_CONFIGURATION = 'IDENTITY'
 
 
-def assume_identity():
+def get_identity_name(identity_kind=GLOBAL_APPLICATION_CONFIGURATION):
+    return os.environ.get(identity_kind, 'dev/beanstalk/cgap-dev')
+
+
+def get_identity_secrets(identity_kind=GLOBAL_APPLICATION_CONFIGURATION) -> dict:
```

Comment:
I would map this to `assume_identity` to avoid the incompatible change while we track all the usage down.

Reply:
I think there is an 
```
assume_identity = get_identity_secrets
```
farther down that does exactly this.